### ISSUE TITLE
Can't edit question although can delete it and duplicate it

### ISSUE THREAD

**Issue by quadrigal** [fpcorso/quiz_master_next#341]
Curious one this one. When I click on "Edit" to edit a question nothing happens. I can however "Delete" or "Duplicate" without problems. I'm using Chrome in Win 10 but repeated the problem with IE. I also have a similar problem with the Results Page tab in that I can add a new page but I cannot delete it- Nothing happens when I click "Delete".

**Comment by fpcorso** [comment]
Hey @quadrigal,

Thank you for choosing to use this plugin! This sounds like a JavaScript issue. Can you perform a plugin check to rule out plugin conflicts?

This process will check to see if there is a plugin that you have installed that may not be compatible with the plugin that you are trying to use. To do this, first de-activate all other plugins besides Quiz And Survey Master. Now, check to see if it is working correctly. If this fixed the problem, this suggests that it is a plugin conflict issue. To discover which plugin may contain the conflict, activate one plugin and then check if the plugin you are setting up still works. If it no longer works again, the plugin you just re-activate is the plugin conflict. If not, continue to activate the plugins one by one and check to see if the problem arises again. If you do find a plugin conflict, please let me know which plugin and which version of the plugin that you are using.

**Comment by quadrigal** [comment]
Hi- just to let you know that I did find out the problem in the end. I had protected my Wordpress installation using an htaccess file and this apparently broke the some functionality (Ajax?). Anyway I fixed it by adding another .htaccess in the wp-admin directory and adding some code as described here:

http://www.wpbeginner.com/wp-tutorials/how-to-password-protect-your-wordpress-admin-wp-admin-directory/

All works fine now!
